test/integration/local_lambda/test_end_to_end.py

```diff
@@ -72,20 +72,34 @@ def tagged_name(self, name, architecture):
 
     def get_tag(self, architecture):
         return "" if architecture == "" else str(f"-{architecture}")
+    
+    def run_command(self, cmd):
+        Popen(cmd.split(" ")).communicate()
+    
+    def sleep_1s(self):
+        time.sleep(SLEEP_TIME)
+        
+    def invoke_function(self, port):
+        return requests.post(
+            f"http://localhost:{port}/2015-03-31/functions/function/invocations", json={}
+        )
```

Comment:
nit: I don't know if breaking the main method to its granularity adds or removes from the readability of `create_container_and_invoke_function` method. 

Please note that this is a nit.